<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransDataToRadioDlg</class>
 <widget class="QDialog" name="TransDataToRadioDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>254</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TransDataToRadioDlg</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>361</width>
     <height>162</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>发射率文件：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>输出文件：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_TempFile"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>温度文件：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_Output"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_TransRatio"/>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>um</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>起始波长：</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_StartWave"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>终止波长：</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_EndWave"/>
    </item>
    <item row="4" column="2">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>um</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="pushButton_Temp">
      <property name="text">
       <string>选择</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="pushButton_TransRatio">
      <property name="text">
       <string>选择</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="pushButton_Output">
      <property name="text">
       <string>选择</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_Do">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>210</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>执行</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Progress">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>进度: 0%</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
